ossec_security/datadog_checks/ossec_security/data/conf.yaml.example

```diff
@@ -0,0 +1,20 @@
+## Log Section
+##
+## type - required - Type of log input source (tcp / udp / file / windows_event).
+## port / path / channel_path - required - Set port if type is tcp or udp.
+##                                         Set path if type is file.
+##                                         Set channel_path if type is windows_event.
+## source  - required - Attribute that defines which integration sent the logs.
+## encoding - optional - For file specifies the file encoding. Default is utf-8. Other
```

Comment:
```suggestion
## encoding - optional - Specifies the file encoding. Default is utf-8. Other
```

Reply:
Tried doing changes as suggested but github checks are failing seems it don't allow modification in conf.yaml.exmaple.
https://github.com/DataDog/integrations-core/actions/runs/9970284776/job/27548878494?pr=17938

hence reverted this change for now.